presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -118,6 +118,13 @@ String Functions
     ``entryDelimiter`` splits ``string`` into key-value pairs. ``keyValueDelimiter`` splits
     each pair into key and value.
 
+.. function:: split_to_multimap(string, entryDelimiter, keyValueDelimiter) -> map<varchar, array<varchar>>
+
+    Splits ``string`` by ``entryDelimiter`` and ``keyValueDelimiter`` and returns a map,
```

Comment:
nit: returns a multimap

Reply:
Let's change this to (basically copied from above):
```
Splits ``string`` by ``entryDelimiter`` and ``keyValueDelimiter`` and returns a map
containing an array of values for each unique key. ``entryDelimiter`` splits ``string``
into key-value pairs. ``keyValueDelimiter`` splits each pair into key and value. The
values for each key will be in the same order as they appeared in ``string``.
```

Add a note to `split_to_map` that mentions the behavior for duplicate keys. (I don't know if we take the first value for a key or if it is arbitrary. If it is the first value, we should specify that, and make sure there are tests which cover it.)

Reply:
`split_to_map` would fail if there are duplicates. The behavior is tested in the test for `split_to_map`.